Lesson21/bilge/todo-app/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>To-Do App</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+  </head>
+  <body>
+    <main class="container">
+      <h1>To-do List</h1>
+      <h2>You may add your to-do items below:</h2>
```

Comment:
Move this text to the label element instead. It's very important that all input have descriptive label text associated with them.